Don't reset target when reading included rpmrc's. Doesn't seem to break
anything and fixes rhbz#232429...

diff -up rpm-4.4.2.2/lib/rpmrc.c.noreset rpm-4.4.2.2/lib/rpmrc.c
--- rpm-4.4.2.2/lib/rpmrc.c.noreset	2008-01-11 10:30:12.000000000 +0200
+++ rpm-4.4.2.2/lib/rpmrc.c	2008-01-11 10:15:27.000000000 +0200
@@ -641,7 +641,9 @@ static int doReadRC( /*@killref@*/ FD_t 
 		while (*se && !xisspace(*se)) se++;
 		if (*se != '\0') *se++ = '\0';
 
+#if 0
 		rpmRebuildTargetVars(NULL, NULL);
+#endif
 
 		fn = rpmGetPath(s, NULL);
 		if (fn == NULL || *fn == '\0') {
